src/ui-kit/Button/Button.mobile.tsx

```diff
@@ -39,6 +39,9 @@ export const Button = JSX<ButtonWithIconProps>(
     ariaLabel,
     ...rest
   }) => {
+    // its non valid HTML attribute. Must be excluded from "rest"
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    const { asSVG, iconRight, appendRight, icon, ...usedProps } = rest;
```

Comment:
```suggestion
    const { asSVG, iconRight, appendRight, icon, ...buttonInnerRest} = rest;
```

Или buttonInnerProps, точно не usedProps